src/ctia/http/handler.clj

```diff
@@ -87,7 +88,7 @@
           `(context
             ~(:route-context entity) []
             :tags ~(:tags entity)
-            (entity->routes (~(:entity entity) entities) ~gsm)))))))
+            (entity->routes (~(:entity entity) entities/entities) ~gsm)))))))
```

Comment:
There are two `entities` in this scope, one local, one global. Being explicit for the global one here to avoid accidents.